### PR TITLE
Security upgrade in routes.py

### PR DESCRIPTION
**Pull request by Deeptig9138**
Input Validation Functions: Created separate functions to validate the username, email, and password format. This ensures that the input adheres to specific rules before further processing.
Input Sanitization with Bleach: Sanitized the chatbot name and prompt using bleach.clean() to prevent any potential XSS attacks. This is crucial for any user-generated content that will be rendered in the HTML.
User Query Sanitization: Similarly sanitized the user query input in the chatbot interaction method.
Feedback on Invalid Inputs: Provided user feedback for invalid inputs in the login and signup routes.
Use of Regular Expressions: Regular expressions are utilized to enforce constraints on usernames and emails, ensuring they are in the correct format.

### CODE REVIEW

**Comment by Deeptig9138** [comment]
done

File: routes.py [hunk 1]
```diff
@@ -132,79 +90,6 @@ def create_chatbot():
 
         return render_template("create_chatbot.html")
 
-    @app.route("/chatbot/<int:chatbot_id>/update", methods=["GET", "POST"])
```
Comment (kom-senapati) [inline routes.py:135]: Dont delete this route

File: routes.py [hunk 2]
```diff
@@ -88,37 +62,21 @@ def signup():
                 return redirect(url_for("login"))
             except IntegrityError:
                 db.session.rollback()
-                flash("Username or email already exists.","signup-error")
+                flash("Username or email already exists.", "signup-error")
 
         return render_template("signup.html")
 
-    @app.route("/logout")
```
Comment (kom-senapati) [inline routes.py:95]: Dont delete logout

File: routes.py [hunk 3]
```diff
@@ -3,72 +3,40 @@
...
+def validate_email(email):
+    return re.match(r'^[^@]+@[^@]+\.[^@]+$', email) is not None
+
+def validate_password(password):
+    return len(password) >= 6  # Ensure password length is at least 6 characters
 
 def register_routes(app, db, bcrypt):
     @app.route("/")
     def index():
-        if Chatbot.query.count() == 0:
```
Comment (kom-senapati) [inline routes.py:11]: Dont delete this if part

File: routes.py [hunk 4]
```diff
@@ -3,72 +3,40 @@
 from models import User, Chatbot, Chat
 from sqlalchemy.exc import IntegrityError
 from ai import chat_with_chatbot
+import bleach  # Importing bleach for sanitizing inputs
```
Comment (kom-senapati) [inline routes.py:6]: Remove all these comments we dont need comments

Comment (kom-senapati) [reply routes.py:6]: Also add this bleach to requirements.txt

**Review by kom-senapati — CHANGES_REQUESTED** [review]
Do these things and commit!

**Comment by kom-senapati** [comment]
Its been 2 days give updates

**Comment by kom-senapati** [comment]
Are you working @Deeptig9138 ?

**Pull request closed without merging**